packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -364,6 +364,20 @@ new cognito.UserPool(this, 'myuserpool', {
 
 ```
 
+Sending emails through SES verified domain, you could use the prop `sesVerifiedDomain` and pass your verified domain value.
```

Comment:
This comment should better explain the use case for this feature, and should mention the use case from the linked issue.